packages/react/src/experimental/Widgets/Content/ListItems/WidgetAvatarsListItem/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { cn } from "../../../../../lib/utils"
+import { AvatarVariant } from "../../../../Information/Avatars/Avatar"
+import { AvatarList } from "../../../../Information/Avatars/AvatarList/index.tsx"
+import { EmojiAvatar } from "../../../../Information/Avatars/EmojiAvatar"
+
+type Props<Id extends string | number = string | number> = {
+  id: Id
+  emoji: string
+  title: string
+  subtitle?: string
+  avatars: AvatarVariant[]
+  onClick?: (id: Id) => void
+}
+
+export type WidgetAvatarsListItemProps<
+  Id extends string | number = string | number,
+> = Props<Id>
+
+type WrapperProps = {
+  onClick?: (ev: React.MouseEvent<HTMLAnchorElement>) => void
+  className: string
+  children: React.ReactNode
+}
+
+const Wrapper: React.FC<WrapperProps> = ({ onClick, className, children }) => {
+  return onClick ? (
+    <a className={className} onClick={onClick} tabIndex={0}>
+      {children}
+    </a>
+  ) : (
+    <div className={className} tabIndex={-1}>
+      {children}
+    </div>
+  )
+}
+
+export function WidgetAvatarsListItem({
+  id,
+  title,
+  subtitle,
+  emoji,
+  avatars,
+  onClick,
+}: Props) {
+  const className = cn(
+    "flex flex-row gap-2 rounded-md border border-solid border-transparent p-2 text-f1-foreground",
+    onClick
+      ? "cursor-pointer hover:bg-f1-background-tertiary focus:border-f1-background-selected-bold focus:outline-none"
+      : undefined
+  )
+  const handleOnClick = (ev: React.MouseEvent<HTMLAnchorElement>) => {
+    ev.preventDefault()
+    onClick?.(id)
+  }
+
+  const computedSubtitle = subtitle || `${avatars.length} people`
```

Comment:
we don't use default text in F1, maybe better request a function for subtitle that receives an amount in a param: 

```
<WidgetAvatarsListItem subtitle={(length) => t('whatever', {count: length})} />
```

Reply:
solved, I'm using the subtitle prop now, thank you ❤️